assets/queries/ansible/s3_bucket_rules_with_awskms/query.rego

```diff
@@ -0,0 +1,40 @@
+package Cx
+
+CxPolicy [ result ] {
+   document := input.document[i]
+   tasks := getTasks(document)
+   task := tasks[t]
+   s3_bucket := task["amazon.aws.s3_bucket"]
+   encryption := s3_bucket.encryption
+   
+   not encryption == "AES256"
+   check_key_id(s3_bucket)
```

Comment:
The query description leads me to think that when the encryption algorithm is AES256, the master key should be empty/undefined, but you are not verifying it. Should you be?